iFixit/react-commerce
# Refactor all remaning product page sections

closes #1507 

With this PR all sections are now managed by the product model. Next step will be to attach strapi as a source to allow customization of featured product pages

## QA

1. Open [product page preview](https://react-commerce-git-refactor-all-product-page-sections-ifixit.vercel.app/products/iphone-6s-screen)
2. Verify that sections still render as in production (with some minor background adjustments)
